src/metaxy/data_versioning/calculators/duckdb.py

```diff
@@ -89,23 +88,23 @@ def _load_extensions(self) -> None:
 
         for ext_spec in self.extensions:
             if isinstance(ext_spec, str):
-                # Simple string form - install from community repo
                 ext_name = ext_spec
-                # Install and load extension
-                backend.raw_sql(f"INSTALL {ext_name}")
-                backend.raw_sql(f"LOAD {ext_name}")
+                ext_repo = "community"
+            elif isinstance(ext_spec, Mapping):
+                ext_name = str(ext_spec.get("name", ""))
+                ext_repo = str(ext_spec.get("repository", "community"))
             else:
-                # Dict form with optional repository
-                ext_name = ext_spec.get("name", "")
-                ext_repo = ext_spec.get("repository", "community")
+                ext_name = str(getattr(ext_spec, "name", ""))
```

Comment:
no reason to use `getattr` here, should do better type narrowing instead

Reply:
not the latest version - should be fixed now